03_ScienceExamples/MagellanicClouds/ExtremelyMetalPoorStarsInTheLMC.ipynb

```diff
@@ -0,0 +1,962 @@
+{
```

Comment:
<p>I now understand why both DR3 and DR4 are mentioned, but this only became clear after reading the last sentence of the summary. You might consider explicitly noting this in the Goal section for clarity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/260/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>